src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -65,24 +79,51 @@ export default function NurseryWithdrawals(): JSX.Element {
     { key: 'hasReassignments', name: '', type: 'string' },
   ];
 
+  const filterColumns = useMemo<FilterField[]>(
+    () => [
+      { name: 'purpose', label: strings.PURPOSE, type: 'multiple_selection' },
+      { name: 'facility_name', label: strings.FROM_NURSERY, type: 'multiple_selection' },
+      { name: 'destinationName', label: strings.DESTINATION, type: 'multiple_selection' },
+      { name: 'batchWithdrawals.batch_species_scientificName', label: strings.SPECIES, type: 'multiple_selection' },
+      { name: 'withdrawnDate', label: strings.WITHDRAWN_DATE, type: 'date_range' },
+    ],
+    []
+  );
+
+  const [filterOptions, setFilterOptions] = useState<FieldOptionsMap>({});
+
+  useEffect(() => {
+    const getApiSearchResults = async () => {
+      setFilterOptions(await NurseryWithdrawalService.getFilterOptions(selectedOrganization.id));
+    };
+    getApiSearchResults();
+  }, [selectedOrganization]);
+
+  const filterPillData = useMemo(
+    () =>
+      Object.keys(filters).map((key) => {
+        const removeFilter = (k: string) => {
+          const result = { ...filters };
+          delete result[k];
+          setFilters(result);
+        };
+
+        return {
+          id: key,
+          label: filterColumns.find((f) => key === f.name)?.label ?? '',
+          value: filters[key].values.join(', '),
```

Comment:
just curious, why join?

Reply:
This combines the values into a comma-separated list in the case of multiple values selected by the filter. Example:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/114949086/219793411-4f9b72f7-c5f3-466a-9ec8-aa0929fefe02.png">

Reply:
oh yes, ofc.. thanks 🙏